File: client/iih_stub/iih.ci_stub/iih.ci.ord/dto/d/LabDTO.cs
using System;
using xap.mw.core.data;
using xap.mw.coreitf.d;

namespace iih.ci.ord.dto.d
{
   public class LabDTO :BaseDTO
    {
        //private String id_rptlab, id_srv, val_rstrptlab;
        //private FDateTime dt_rptlab;

      
        public string Id_rptlab
        {
            get { return getAttrVal<string>("Id_rptlab", null); }
            set { setAttrVal<string>("Id_rptlab", value); }
        }

      
        public string Id_srv
        {
            get { return getAttrVal<string>("Id_srv", null); }
            set { setAttrVal<string>("Id_srv", value); }
        }

    
        public string Val_rstrptlab
        {
            get { return getAttrVal<string>("Val_rstrptlab", null); }
            set { setAttrVal<string>("Val_rstrptlab", value); }
        }


        public DateTime? Dt_rptlab
        {
            get { return getAttrVal<FDateTime>("Dt_rptlab", null); }
            set { setAttrVal<FDateTime>("Dt_rptlab", value); }
        }
        public string Sd_restrptlabtp
        {
            get { return getAttrVal<string>("Sd_restrptlabtp", null); }
            set { setAttrVal<string>("Sd_restrptlabtp", value); }
        }
      


    }
}

File: server/iih_pub/iih.ci_pub/src/main/java/iih/ci/ord/dto/d/LabDTO.java
package iih.ci.ord.dto.d;

import xap.mw.core.data.BaseDTO;
import xap.mw.coreitf.d.FDateTime;

public class LabDTO extends BaseDTO{
	
	
	

	public String getId_rptlab() {
		return ((String) getAttrVal("Id_rptlab"));
	}
	public void setId_rptlab(String Id_rptlab) {
		setAttrVal("Id_rptlab", Id_rptlab);
	}
	public String getId_srv() {
		return ((String) getAttrVal("Id_srv"));
	}
	public void setId_srv(String Id_srv) {
		setAttrVal("Id_srv", Id_srv);
	}
	public String getVal_rstrptlab() {
		return ((String) getAttrVal("Val_rstrptlab"));
	}
	public void setVal_rstrptlab(String Val_rstrptlab) {
		setAttrVal("Val_rstrptlab", Val_rstrptlab);
	}
	
	public String getSd_restrptlabtp() {
		return ((String) getAttrVal("Sd_restrptlabtp"));
	}
	public void setSd_restrptlabtp(String Sd_restrptlabtp) {
		setAttrVal("Sd_restrptlabtp", Sd_restrptlabtp);
	}
	public FDateTime getDt_rptlab() {
		return ((FDateTime) getAttrVal("Dt_rptlab"));
	}
	public void setDt_rptlab(FDateTime Dt_rptlab) {
		setAttrVal("Dt_rptlab", Dt_rptlab);
	}
	
	

}
